.travis.yml

```diff
@@ -4,9 +4,11 @@ rbenv:
 addons:
   postgresql: 9.6
   chrome: stable
-dist: trusty
+dist: xenial
 script:
```

Comment:
added more to the travis script because this is what travis performs and somehow i didnt have trusty changed

Reply:
Wait, you had to change it back to xenial? Didnt we update it the other day?